pygmt/src/meca.py

```diff
@@ -221,6 +221,50 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
     r"""
     Plot focal mechanisms.
 
+    The following focal mechanism conventions are supported:
+
+    .. list-table:: Supported focal mechanism conventions.
+       :widths: 15 15 40 30
+       :header-rows: 1
+
+       * - Convention
+         - Description
+         - Focal parameters
+         - Remark
+       * - ``"aki"``
+         - Aki and Richard
+         - *strike*, *dip*, *rake*, *magnitude*
+         - angles in degrees
+       * - ``"gcmt"``
+         - global centroid moment tensor
+         - | *strike1*, *dip1*, *rake1*,
+           | *strike2*, *dip2*, *rake2*,
+           | *mantissa*, *exponent*
+         - | angles in degrees;
+           | *mantissa*, *exponent*are for
+           | seismic moment in dyne-cm
+       * - ``"mt"``
+         - seismic moment tensor
+         - | *mrr*, *mtt*, *mff*,
+           | *mrt*, *mrf*, *mtf*,
+           | *exponent*
+         - | moment components
+           | in dynes-cm
```

Comment:
```suggestion
         - | moment components 
           | in :math:`10*{exponent}` dyne-cm
```

Reply:
Jupp! Actually, there are several versions used: "dynes-cm", "dyne-cm", and "dyn-cm" with and without the hyphen 🙃. I prefer “dyn”. Personally, I do not like this hyphen because it's stickly speaking a multiplication between `dyn` and `cm` (same as for `10^-7 N m`).